style.css

```diff
@@ -116,3 +116,365 @@
   gap: 19px;
   align-items: baseline;
 }
+
+/* #work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+} */
```

Comment:
- [ ] It's best if you don't commit old code as inline comments, they will make your project look messy. If you need to review a previous version of your code, you can always use git history